### PR TITLE
fix(docs/kiosk): update some Kiosk links, add more examples

### LINKED ISSUE #4020: [kiosk] Update Kiosk related docs(links, examples, etc)
Update Kiosk related documentation:
1. Links
2. Examples
3. Feature descriptions

### PR DESCRIPTION
**Pull request by Dkwcs**
# Description of change

Updating kiosk documentation with some links and brand-new examples.
Links refer to the develop branch in advance.

## Links to any relevant issues

Fix #4020 .

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Fix

## How the change has been tested

`pnpm iota-docs dev`

### CODE REVIEW

**Review by vivekjain23 — APPROVED** [review]

**Comment by vivekjain23** [comment]
Check the formatting of the docs; the rest is looking good.

**Review by miker83z — APPROVED** [review]

File: docs/content/developer/iota-101/nft/marketplace.mdx [hunk 1]
```diff
@@ -0,0 +1,218 @@
+---
+description: A brief introduction to implementing NFT marketplace extension using the Kiosk Apps standard in IOTA's Move language.
+---
+
+# Marketplace Extension Usage
+
```
Comment (miker83z) [inline docs/content/developer/iota-101/nft/marketplace.mdx:5]: Could you add a brief introduction description to this page? In a similar fashion of what you find at the top of `rent-nft.mdx`

Comment (Dkwcs) [reply docs/content/developer/iota-101/nft/marketplace.mdx:5]: Done

**Review by lucas-tortora — APPROVED** [review]
LGTM :) Just one note: We can import the README file from the example folder as in https://github.com/iotaledger/iota/blob/develop/docs/content/operator/validator-operation/docker/README.mdx?plain=1#L4 .

**Comment by Dkwcs** [comment]
> LGTM :) Just one note: We can import the README file from the example folder as in https://github.com/iotaledger/iota/blob/develop/docs/content/operator/validator-operation/docker/README.mdx?plain=1#L4 .

Nice feature, thanks

**Pull request merged**